src/transformers/models/dpt/configuration_dpt.py

```diff
@@ -179,9 +180,6 @@ def __init__(
         self.hidden_size = hidden_size
         self.is_hybrid = is_hybrid
 
-        if use_pretrained_backbone:
-            raise ValueError("Pretrained backbones are not supported yet.")
```

Comment:
Will the logic change here? In `verify_backbone_config_arguments` we have 
```python
if not use_timm_backbone and use_pretrained_backbone:
        raise ValueError(
            "Loading pretrained backbone weights from the transformers library is not supported yet. `use_timm_backbone` must be set to `True` when `use_pretrained_backbone=True`"
        )
```

Reply:
Yes! For DPT this is because it uses a HF backbone by default and we don't support HF pretrained backbones yet. There's a PR to add this here: #31145